### PR TITLE
added battery percentage and state

### PR DESCRIPTION
**Pull request by rishab2890**
Added battery percentage and battery status using broadcastreciever and batteryManager
Screenshot

### CODE REVIEW

**Comment by noman2002** [comment]
hey, @rishab2890  Add discription to it. Attach screenshot .

**Review by Areeb786123 — CHANGES_REQUESTED** [review]
U have implement only charging  and its percentage without canvas and also Implement scrollView .

File: README.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ This repository consist of some custom views and UI components for android
 * [CustomAnalogClockView](app/src/main/java/com/example/customviewdemoapplication/Views/CustomAnalogClock.java)
 
 # Screenshot
-<img src="screenshots/ss2.png">
+<img src="screenshots/Screenshot_1601744805.png">
```
Comment (danishjamal104) [inline README.md:14]: replace the existing screenshot without changing the file name.

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -13,6 +13,22 @@
     tools:context=".MainActivity"
     android:orientation="vertical">
 
+    <TextView
```
Comment (danishjamal104) [inline app/src/main/res/layout/activity_main.xml:16]: You have to implement a custom view. Don't use predefined tags like textView or anything define your own View class BatteryView or something.

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -59,7 +75,6 @@
         android:layout_height="115dp"
         app:text="Hello"
         android:strokeColor="@color/colorPrimary"
```
Comment (danishjamal104) [inline app/src/main/res/layout/activity_main.xml:77]: If you are making any changes or removing any attribute then specify it in the description.

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -79,6 +94,8 @@
         android:layout_height="150dp" />
 
 
+
```
Comment (danishjamal104) [inline app/src/main/res/layout/activity_main.xml:97]: Please avoid adding an extra line in files.
Also, try to remove EOF.

**Review by danishjamal104 — CHANGES_REQUESTED** [review]
@rishab2890 please look into all the comments and make necessary changes.

**Review by noman2002 — CHANGES_REQUESTED** [review]
Make changes as suggested by @danishjamal104

**Comment by noman2002** [comment]
@rishab2890 please be active on the PR until it is open, else I have to close this PR.

**Pull request closed without merging**